mongo/readpref/readpref.go

```diff
@@ -97,3 +107,27 @@ func (r *ReadPref) Mode() Mode {
 func (r *ReadPref) TagSets() []tag.Set {
 	return r.tagSets
 }
+
+// Hedge returns the document used to configure server hedged reads. If no hedge options were specified when creating
+// the read preference, nil is returned.
+func (r *ReadPref) Hedge() bson.Raw {
```

Comment:
It seems cleaner, and more consistent with the other properties of `ReadPref`, to instead expose a  `HedgeEnabled` property of type `*bool`.  And then leave it to `Operation.createReadPref` to figure out how that gets serialized, as is done for all the other read preference properties.